net/qpid-proton/Portfile

```diff
@@ -19,14 +19,14 @@ homepage            https://qpid.apache.org
 
 PortGroup           github 1.0
 
-github.setup        apache qpid-proton fa80534c87940b44fefd36abff4c5a9a79b99f47
-version             0.18.1
-
-checksums           rmd160  61841dd0416cc738b02e5137316bf9fd46821ced \
-                    sha256  d762c7b07ed6092ebdbcc49da731425bc5a71bef084fd9c29a0e26e6e18f275b
+github.setup        apache qpid-proton 0.19.0
+version             0.19.0
 
```

Comment:
good news! github.setup will automatically set the version for you as well, so you don't need this line.

Reply:
Thanks for the review and good to know! Will update.